api/types/license.go

```diff
@@ -27,39 +27,45 @@ import (
 // License defines teleport License Information
 type License interface {
 	Resource
+
 	// GetReportsUsage returns true if teleport cluster reports usage
 	// to control plane
 	GetReportsUsage() Bool
+	// SetReportsUsage sets usage report
```

Comment:
```suggestion
	// SetReportsUsage toggles usage report
```

Reply:
I think "sets" is more correct here as "toggle" usually means inverting the state, while set sets the provided value.